marketplaceItems/audioFocus/appResources/appData/audioFocus_app.js

```diff
@@ -0,0 +1,459 @@
+(function () {
+
+    /*
+        Solo Point
+        Created by Milad Nazeri on 2019-01-07
+        Copyright 2019 High Fidelity, Inc.
+    
+        Distributed under the Apache License, Version 2.0.
+        See the accompanying file LICENSE or http://www.apache.org/licenses/LICENSE-2.0.html
+
+        Point to solo someone to hear them better in a crowd!
+        
+    */
+
+
+    // *************************************
+    // START UTILITY FUNCTIONS
+    // *************************************
+    // #region Utilty
+
+    var debug = true;
+    var FALSE = "false";
+
+    // Easy log function checking for message, an object to stringify, and whether it should be enabled or not
+    function log(message, object, enabled) {
+        if (!debug || enabled === FALSE) {
+            return;
+        }
+
+        var finalMessage;
+
+        finalMessage = "\n\t" + message + ":" + "\n";
+
+        if (typeof object !== 'undefined') {
+            finalMessage += "\n\t\t" + JSON.stringify(object, null, 4) + "\n";
+        }
+
+        print(finalMessage);
+    }
+
+
+    // Utility function for the ControllerWorldLocation offset 
+    function getGrabPointSphereOffset(handController, ignoreSensorToWorldScale) {
```

Comment:
Please make a comment here that this function is a modified version of what's found in `scripts/system/libraries/controllers.js`

Reply:
`ignoreSensorToWorldScale` is never used as an argument, please remove